config/zones.json

```diff
@@ -115,6 +115,16 @@
       "production": "BG.fetch_production"
     }
   },
+  "BO": {
+    "capacity": {
+      "hydro": 483.22,
+      "unknown" : 1344.55,
+      "wind": 27 
+    },
+    "parsers": {
+      "production": "BO.fetch_production"
```

Comment:
good catch. I'll add it for Denmark.
The key should be "generationForecast", so the line to add is 
`"generationForecast": ""BO.fectch_generation_forecast"`